.devcontainer/gatekeeper.http

```diff
@@ -8,8 +8,8 @@ Content-Type: application/json
   "kind":"ProviderRequest",
   "request": {
     "keys": [
-      "wabbitnetworks.azurecr.io/test/notary-image:signed",
-      "wabbitnetworks.azurecr.io/test/notary-image:unsigned"
+      "wabbitnetworks.azurecr.io/test/notation-image:signed",
```

Comment:
We already migrate the test image from ACR to ghcr, we can use `ghcr.io/deislabs/ratify/notary-image:signed` instead.